docs/content/Configuration/Config.md

```diff
@@ -577,6 +578,10 @@ module.exports = {
 };
 ```
 
+### cacheAndQueueDriver
+
+The cache and queue driver to use for the Cube.js deployment. Defaults to `redis`.
```

Comment:
> Defaults to `redis`.

It's not true. By default, it's a memory for not production env.

Reply:
```suggestion
The cache and queue driver to use for the Cube.js deployment. Defaults to
`memory` in development, `redis` in production.
```